app/models/chargeback.rb

```diff
@@ -1,4 +1,6 @@
 class Chargeback < ActsAsArModel
+  singleton_class.send(:prepend, DescendantLoader::ArDescendantsWithLoader)
```

Comment:
@jrafanie This was the failure when he had `DescendantLoader.instance.load_subclasses(self)` https://travis-ci.org/ManageIQ/manageiq/jobs/132351191

```
load_missing_constant': Circular dependency detected while autoloading constant ChargebackContainerProject (RuntimeError)
```